torchacc/utils/patch.py

```diff
@@ -97,8 +97,19 @@ def _flash_attention_forward(
                 window_size = (sliding_window,
                                sliding_window) if use_sliding_windows else (-1,
                                                                             -1)
-                # TODO(to TianXing): support position_ids
-                if attention_mask is not None:
+                if position_ids is not None:
+                    return ops.flash_attn_varlen_position_ids_xla(
+                        query_states.contiguous(),
```

Comment:
.contiguous() for q,k,v can be removed since we have call `maybe_contiguous` in flash_attn_varlen_position_ids_xla